### DISCUSSION TITLE
Freeplane 1.11.1 released as stable version

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#1123] [Announcements]
Many thanks to all community members who contributed to feature definition, discussions, testing, translations, and so on.
We make this project together.

Enjoy.
Dimitry

[Changelog](https://www.freeplane.org/info/history/history_en.txt)

**Comment by sifran-github** [comment]
Amazing work! 👍 🚀

**Reply by MMAcode** [reply]
Hi folks, I saw various layouts introduced as per image...
<img width="1020" alt="image" src="https://github.com/freeplane/freeplane/assets/52301171/a5f9ba6a-d38f-426a-86e2-fe25214fee9e">
...
and I found 2 options to turn some layout features on/off as per next image...
<img width="1244" alt="image" src="https://github.com/freeplane/freeplane/assets/52301171/680a8fa3-d268-47c2-9cc5-8cabed44f0f3">

...but selecting any of those options seems to make no changes at all.

**Do we have any docs or videos as to how to use these layouts?** (And is it possible they are not working on my M1 MAC book?)

Thanks!

**Comment by biswajit-goswami** [comment]
I am also not able to figure out the new layout features..any explanation doc or video ?

**Reply by biswajit-goswami** [reply]
I got it..
![image](https://github.com/freeplane/freeplane/assets/11291623/566032a9-54e7-4db3-8baf-54f2f6c01418)
 and then 
![image](https://github.com/freeplane/freeplane/assets/11291623/ea6117a1-c921-4f89-9c79-59f651035bc4)

wow...few years back I had requested this [here](https://sourceforge.net/p/freeplane/discussion/758437/thread/a7543d0f44/?limit=25#9ae7)..and finally its implemented...thank you.

**Reply by dpolivaev** [reply]
I have implemented a new menu item with the same functionality for Freeplane 1.11.3

Format->Children nodes layout

**Reply by dpolivaev** [reply]
And also improved the latest features map to show the above information, and also added it to the help menu so that user can check it any time later

**Comment by EdoFro** [comment]
Hi all,
I want to congratulate and thank @dpolivaev  for the great job that he achieved with this version. 
The new layouts option wasn’t something easy to define. Surely it was hard to code, so it works with all other existing features and without loosing speed or capacity. 

In my opinion, Dimitry exceeded in quality and effort doing his part of the job. 

Now it’s time for the rest of us, specially the ones that asked for the new features, to update the documentation and add a guide to use them, so the rest of the community can benefit from this great software. 
jm2c,
edo

**Comment by Illegit** [comment]
I'm not a creator nor either a programmer but just a user. Very nice solutions, I've put them immediately at work. Thank 
you Dimitry, and thankyou all.

**Comment by ldmpub** [comment]
I give it a try today, this new feature is just amazing 👏 ... I was able to quickly present nodes in an "[organigramm](https://www.business-wissen.de/res/images/Abbildung-9901109-a.png)" layout in few minutes.

Thanks again to you all and a big hug to @dpolivaev 🙏.
